### DISCUSSION TITLE
Language problem and score not showing

### DISCUSSION THREAD

**Discussion by Brown2222** [IBAX-io/go-ibax#873] [Q&A] [answered]
### Describe the bug

When I translate Turkish from the browser, the score does not appear, everything seems to be zero.

### Reproduction

![Screenshot_20220126_100532_com android chrome](https://user-images.githubusercontent.com/98290360/151118218-4f9ba63d-a537-423e-a136-14a30908c967.jpg)

### System Info

```shell
Browers
```

### Logs

```shell
Go-ibax
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ibaxio** [comment, marked as answer]
Please refer to this discussion: https://github.com/IBAX-io/go-ibax/discussions/453#discussioncomment-2031685
